src/Core/Framework/Api/HealthCheck/Model/Result.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Shopware\Core\Framework\Api\HealthCheck\Model;
+
+use InvalidArgumentException;
+
+class Result
```

Comment:
Would be nice for checks to maybe add arbitrary data, for example database and cache could add connection details or adapter, eg array or redis for cache. This could be included in response.